pxtsim/simdriver.ts

```diff
@@ -390,6 +390,21 @@ namespace pxsim {
             }
         }
 
+        private postMessageCore(frame: HTMLIFrameElement, msg: SimulatorMessage) {
+            frame.contentWindow.postMessage(msg, frame.dataset['origin']);
+
+            if (U.isLocalHost() && (pxt as any)?.appTarget?.id) {
```

Comment:
```suggestion
            if (U.isLocalHostDev() && (pxt as any)?.appTarget?.id) {
```

to filter out electron

Reply:
good call, but i need to port over that function into pxtsim